lib/date/date.ex

```diff
@@ -31,6 +31,7 @@ defmodule Timex.Date do
   @type weekday :: 1..7
   @type weeknum :: 1..53
   @type num_of_days :: 28..31
+  @type quarter :: 1..4
```

Comment:
@bitwalker I'm having second thoughts about creating a new type. Maybe it should be better to make the 'quarter/1' function private and treat the response as a regular integer.